### ISSUE TITLE
Feature request: Source Request

### ISSUE THREAD

**Issue by OmarNazzal150** [ImranR98/Obtainium#1921]
**Describe the feature**
<!-- Add these sources

Xiaomi Mi GetApps

Oppo App Market

TapTap

Accrescent

Cafe Bazaar

Myket

**Additional context**
<!-- Their Offical Website

https://global.app.mi.com

For oppo app store I couldn't find the offical website sorry

https://www.taptap.io/

https://accrescent.app/

https://cafebazaar.ir

https://myket.ir/

**Comment by DwainZwerg** [comment]
Please check previous issues before creating a new one. For Samsung Galaxy Store: https://github.com/ImranR98/Obtainium/issues/1423; for Cafe Bazaar: https://github.com/ImranR98/Obtainium/issues/394; for Myket: https://github.com/ImranR98/Obtainium/issues/395

**Comment by ImranR98** [comment]
Yes in general it's good to have separate issues per source (don't close this one now since it's already open).
Also I don't see why anyone would use Obtainium to download from Accrescent instead of using that app directly (they likely don't have an API we can use anyways).

**Comment by DwainZwerg** [comment]
There are people (like me) who like to have just one app on their device that manages everything. We already support F-Droid and F-Droid third party sources for the same reason. And you don't need Obtainium for APKPure, you can also use their app, etc.

**Comment by jm355** [comment]
Seconding the idea that the reason I use obtainium is to only have one app for managing apps. If I was going to use multiple app stores, I would use f-droid with 3rd party repos, aurora store, and accrescent. I switched to obtainium to be able to uninstall those three stores, although I'm really looking forward to https://github.com/ImranR98/Obtainium/issues/343, since being able to use google play as a source would be way better than other stores like apkpure, aptoide, and uptodown which usually have delayed updates and are generally sketchier imo
